MovieApp/MovieApp/App/DataRepository/MoviesDataRepository.swift

```diff
@@ -0,0 +1,43 @@
+class MoviesDataRepository: MoviesDataRepositoryProtocol {
```

Comment:
Since everything coming from Repo and DS layer is data, there's no need to add it to the name.
There are special cases where models have `Data` in their name, but not the Repo itself.

Reply:
[Changed name](https://github.com/lihter/Movie-App/pull/24/files#diff-119fd471623eca33515c1b8fb62109b006cbeee1e1df1dc1a8e9a2e5ba5d4c27R3) to `MovieRepository`.
Also replaaced the name everywhere in code where it was needed.